contracts/smart-account/interfaces/base/IModuleManager.sol

```diff
@@ -76,6 +76,12 @@ interface IModuleManager {
         uint256 operationLength
     );
 
+    /**
+     * @notice Throws when trying to remove the only enabled module
+     * @param module Module address provided
+     */
+    error CanNotDisableOnlyModule(address module);
```

Comment:
I'll suggest `Cannot` instead of `CanNot` since we already have  `ModuleCannotBeZeroOrSentinel`

Reply:
made both `CanNot`